common/lib/protocol-definitions/src/sockets.ts

```diff
@@ -59,6 +59,11 @@ export interface IConnect {
      * If we have standardized features across all services, they need to be exposed in more structured way.
      */
     supportedFeatures?: Record<string, any>;
+
+    /**
+     * Properties that client can send to server to tell info about client loader, driver versions etc.
```

Comment:
Not been following this work, but do we want to consider a stricter schema here? As opposed to free-form string.

Reply:
Also I noticed in the code some variables misspell correlation as corelation, JFYI

Reply:
Yes it will follow a specific structure as I have mentioned in the comment. We don't want to have pre determined props as that limits extensibility here seeing the usage where client just wants to log the info that it provides as supposed to something which is definitely required by push and vice-versa.